### PR TITLE
Added more tsg templates and quick start templates

### PR DESCRIPTION
**Pull request by ericc1103**

### CODE REVIEW

File: Workbooks/TSG/Dependency Failures TSG/Dependency Failures.workbook [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": true,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Troubleshooting Dependency Failures\nThis troubleshooting guide help you identify failures in your application's dependencies. "
```
Comment (acearun) [inline Workbooks/TSG/Dependency Failures TSG/Dependency Failures.workbook]: help -> helps

File: Workbooks/TSG/Exceptions/Exceptions.workbook [hunk 2]
```diff
@@ -0,0 +1,281 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": true,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Troubleshooting Exceptions\nThis troubleshooting guide help you analyze in your application's exceptions."
```
Comment (acearun) [inline Workbooks/TSG/Exceptions/Exceptions.workbook]: help -> helps

Comment (acearun) [reply Workbooks/TSG/Exceptions/Exceptions.workbook]: take out 'in'

File: Workbooks/TSG/Exceptions/Exceptions.workbook [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+          },
+          {
+            "id": "84caf619-7585-48e0-a9c3-85c364aaa13b",
+            "version": "KqlParameterItem/1.0",
+            "name": "Show",
+            "type": 2,
+            "description": null,
+            "isRequired": true,
+            "multiSelect": null,
+            "query": "datatable(x:string, y:string)[\r\n\"['New Failure Rate (%)'], ['Existing Failure Rate (%)']\", 'New and Existing Failures',\r\n\"['New Failure Rate (%)']\", 'Only New Failures',\r\n\"['Existing Failure Rate (%)']\", 'Only Existing Failures',\r\n]",
```
Comment (acearun) [inline Workbooks/TSG/Exceptions/Exceptions.workbook]: New and Existing Failures -> New and Existing Problems

Comment (acearun) [reply Workbooks/TSG/Exceptions/Exceptions.workbook]: Only New Failures -> Only New Problems

Comment (acearun) [reply Workbooks/TSG/Exceptions/Exceptions.workbook]: Only Existing Failures -> Only Existing Problems

File: Workbooks/TSG/Request Failures/Request Failures.workbook [hunk 4]
```diff
@@ -0,0 +1,264 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": true,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Troubleshooting Request Failures\nThis troubleshooting guide help you identify failures in your application's web requests. "
```
Comment (acearun) [inline Workbooks/TSG/Request Failures/Request Failures.workbook]: help -> helps

File: Workbooks/TSG/Request Failures/settings.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Dependency Failures",
```
Comment (acearun) [inline Workbooks/TSG/Request Failures/settings.json]: Request Failures

File: Workbooks/TSG/Dependency Failures TSG/settings.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Dependency Failures",
+    "author": "Microsoft",
+    "galleries": [{ "type": "tsg", "resourceType": "microsoft.insights/components", "order": 100 }]
```
Comment (acearun) [inline Workbooks/TSG/Dependency Failures TSG/settings.json]: "order": 200

**Comment by ericc1103** [comment]
@acearun Your suggestions have been updated. I guess you would need to approve this again since I made some changes.

**Pull request merged**